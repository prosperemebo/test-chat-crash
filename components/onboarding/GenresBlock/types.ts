import { GenreEnumType } from '@/types'

export interface GenresBlockProps {
    genres: GenreEnumType[]
    onChange: (genre: GenreEnumType) => void
}