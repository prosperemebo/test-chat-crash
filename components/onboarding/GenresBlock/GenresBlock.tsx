import { Wrapper, GenreIconWrapper, GenreButton, Header, ScrollContainer, GridContainer } from './styles'
import { GenreEnumType } from '@/types'
import i18n from '@/localization/i18n'
import { GENRES, GenreConfig } from '@/constants/profile'
import { DisabledItem, SelectedItem } from '@/components/onboarding'
import { GenresBlockProps } from './types'


const GenresBlock = (props: GenresBlockProps) => {
  const { genres, onChange } = props

  const handleGenrePress = (name: GenreEnumType) => () => {
    onChange(name)
  }

  const renderGenres = (item: GenreConfig) => {
    const isItemSelected = genres.includes(item.name)

    //TODO fix translations
    const icon = isItemSelected ? <SelectedItem
      label={item.name}
      itemIcon={item.source}
      itemIcon2={item.source2}
      itemWidth={item.innerWidth}
      itemHeight={item.innerHeight}
    /> : <DisabledItem
      label={item.name}
      itemIcon={item.source}
      itemIcon2={item.source2}
      itemWidth={item.innerWidth}
      itemHeight={item.innerHeight}
    />

    return <GenreButton key={item.name} onPress={handleGenrePress(item.name)}>
      <GenreIconWrapper>
        {icon}
      </GenreIconWrapper>
    </GenreButton>
  }

  return <Wrapper>
    <Header>{i18n.t('profile:favoriteGenres')}</Header>
    <ScrollContainer>
      <GridContainer>
        {GENRES.map(renderGenres)}
      </GridContainer>
    </ScrollContainer>
  </Wrapper>
}

export default GenresBlock