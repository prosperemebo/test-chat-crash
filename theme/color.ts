export const WHITE = '#FFFFFF'
export const t10WHITE = 'rgba(255,255,255,0.1)'
export const t76WHITE = 'rgba(255,255,255,0.76)'

export const RED = '#f60004'
export const RED1 = '#D33E41'


export const BLACK = '#000000'
export const BLACK2 = '#050608'
export const nBLACK = '#15161A'
export const nBLACK10 = '#1F2025'
export const nBLACK30 = '#28292E'
export const nBLACK40 = '#050608'

export const nGRAY10 = '#6B6E79'
export const nGRAY20 = '#858995'
export const nGRAY25 = '#54575e'
export const nGRAY30 = '#2f3137'
export const nGRAY35 = '#52555D'
export const nGRAY40 = '#35363B'


export const tGRAY20 = '#85899599'
export const tGRAY30 = '#545458A6'
export const tGRAY40 = '#45464C'
export const GRAY40 = '#cfcfd0'
export const GRAY45 = '#c9c9cb'
export const GRAY50 = '#aaaaaa'
export const GRAY60 = '#52545D'
export const t40GRAY60 = '#52545D66'
export const GRAY70 = '#383B40'
export const GRAY90 = '#27292d'
export const GRAY100 = '#1f2023'
export const GRAY110 = '#151619'

export const GRAY_ONBOARD = '#8791a5'

export const BLUE = '#50b5e8'
export const BLUE2 = '#0A84FF'

export const PURPLE = '#9A1F99'

export const nBLUE = '#8D88A8'
export const nPURPLE = '#C47AF5'
export const nPURPLE10 = '#873FBB'
export const nPURPLE20 = '#873FBB'
export const nPURPLE40 = '#C882F8'

export const nORANGE10 = '#FF6705'
export const nORANGE20 = '#FF7905'
export const nORANGE40 = '#FFB700'

export const YELLOW2 = '#EBC789'
export const nYELLOW = '#F5D193'

export const nGRAYBACKGROUND2 = '#292A30'
export const nGRAYBACKGROUND3 = '#383940'
export const nGRAYBACKGROUND4 = '#989898'

export const gradient = {
  COLORS: [BLUE, PURPLE],
  START: { x: 0, y: 1 },
  END: { x: 1, y: 0 },
  LABEL_COLOR: WHITE,
}

export const orangeGradient = {
  COLORS: [nORANGE10, nORANGE40],
  START: { x: 0, y: 1 },
  END: { x: 1, y: 0 },
  LABEL_COLOR: WHITE,
}