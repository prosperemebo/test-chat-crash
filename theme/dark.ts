import {
  WHITE,
  BLACK,
  nGRAY10,
  GRAY_ONBOARD,
  nPURPLE,
  nBLUE,
  GRAY40,
  nGRAY20,
  RED1,
  tGRAY20,
  RED,
  nBLACK,
  nORANGE10,
  nORANGE40,
  nPURPLE10,
  GRAY70,
  tGRAY30,
  BLUE2,
  BLACK2,
  nGRAY25,
  nBLACK30,
  GRAY50,
  nBLACK10,
  t10WHITE,
  nGRAYBACKGROUND2,
  GRAY100,
  GRAY60,
  nGRAY40,
  nGRAY30,
  YELLOW2,
  nYELLOW,
  nGRAYBACKGROUND3,
  tGRAY40,
  nGRAY35,
  nBLACK40,
  t40GRAY60,
  nPURPLE20,
  nPURPLE40,
} from './color'

export const dark = {
  tint: nPURPLE,
  primaryTextColor: WHITE,
  primaryBg: nBLACK10,
  headerBg: GRAY100,
  headerBorderColor: GRAY60,
  secondaryTextColor: GRAY60,
  secondaryBgLight: nBLACK30,
  secondaryBg: nBLACK,
  serviceBg: GRAY100,
  inActiveBorderColor2: t10WHITE,
  backgroundColor2: nGRAYBACKGROUND2,
  backgroundColor3: nGRAYBACKGROUND3,
  backgroundColor4: nBLACK10,

  auth: {
    socialIconColor: WHITE,
    socialLoginButton: BLACK,
    dividerColor: nGRAY10,
    noticeText: WHITE,
    forgotPasswordLabel: WHITE,
  },

  onboarding: {
    headerTextColor: WHITE,
    headerStepsTextColor: GRAY_ONBOARD,
    progressBarColor: nPURPLE,
    progressBarBackgroundColor: nBLUE,
    selectedItemTextColor: WHITE,
    disabledItemTextColor: GRAY40,
    gameSectionOverlayColor: nPURPLE,
    gameEmptySectionColor: BLACK,
    gameEmptyIconColor: BLACK2,
  },

  common: {
    labelColor: WHITE,
    infoTextColor: nGRAY20,
    inputIconColor: nGRAY20,
    inputPlaceholderColor: nGRAY20,
    serviceTextColor: WHITE,
    inputTextColor: WHITE,
    inputTextError: RED1,
    inputBorderColor: tGRAY20,
    inputErrorBorderColor: RED,
    inputBackgroundColor: nBLACK,
    toastBackground: nBLACK,
    toastBorderColor: nPURPLE,
    toastTextColor: WHITE,
    buttonGradientOrangeStart: nORANGE10,
    buttonGradientOrangeEnd: nORANGE40,
    buttonGradientPurpleStart: nPURPLE40,
    buttonGradientPurpleEnd: nPURPLE20,
    buttonGradientGrayStart: nGRAY10,
    buttonGradientGrayEnd: nGRAY20,
    buttonDartBackgroundColor: BLACK,
    backButtonColor: nGRAY20,
    switchTextColor: WHITE,
    checkboxIconColor: nPURPLE,
    gradientButtonColorStart: nPURPLE10,
    gradientButtonColorEnd: nPURPLE,
    borderColor: GRAY70,
    genderPickerBorderColor: nGRAY25,
    genderPickerActiveTextColor: WHITE,
    genderPickerTextColor: GRAY50,
    checkboxTextColor: WHITE,
    iconButtonColor: WHITE,
    backgroundColor: nBLACK10,
  },

  settings: {
    socialIconBackgroundColor: nGRAY40,
    premiumFeatureTextColor: nPURPLE,
    buttonIconColor: nGRAY20,
    serviceItemBgColor: nGRAY30,
  },

  postOnboarding: {
    backgroundColor: nBLACK10,
    inputIconColor: nGRAY20,
    inputIconErrorColor: RED1,
    disclosureTextColor: nGRAY20,
    loadingIndicatorColor: BLACK,
  },

  profile: {
    backgroundColor: nBLACK10,
    improveBackgroundColor: YELLOW2,
    improveBannerBackgroundColor: nYELLOW,
    improveBannerTitleColor: nBLACK30,
    coverPictureCardRemoveButtonBg: tGRAY30,
    profilePicturesCountColor: tGRAY40,
  },

  chats: {
    iconColor: nGRAY20,
    inputColor: nBLACK40,
    placeholderColor: nGRAY35,
    appbarBorderColor: t40GRAY60,
    backgroundColor: nBLACK10,
  },

  nav: {
    bottomTabLabelBackgroundColor: nBLACK,
    bottomTabBackgroundColor: nBLACK,
  },

  other: {
    updateApp: {
      textBorderColor: tGRAY30,
      buttonTextColor: BLUE2,
      textColor: WHITE,
    },
  },
}
