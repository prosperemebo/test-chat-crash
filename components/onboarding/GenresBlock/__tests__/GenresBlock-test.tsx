import { render } from '@testing-library/react-native'
import { ThemeProvider } from 'styled-components/native'
import { describe, expect, test } from '@jest/globals'
import GenresBlock from '../'
import { dark } from '@/theme'

describe('<GenresBlock />', () => {

  test('GenresBlock renders correctly', () => {
    const tree = render(
      <ThemeProvider theme={dark}>
        <GenresBlock genres={['RPG']} onChange={() => {}}/>
      </ThemeProvider>,
    ).toJSON()

    expect(tree).toMatchSnapshot()
  })
})