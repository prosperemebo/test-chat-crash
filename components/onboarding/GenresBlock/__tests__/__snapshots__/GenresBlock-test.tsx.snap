// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GenresBlock /> GenresBlock renders correctly 1`] = `
<View
  style={
    {
      "marginBottom": "auto",
      "paddingHorizontal": 21,
    }
  }
>
  <Text
    style={
      {
        "color": "#FFFFFF",
        "fontFamily": "Poppins-Bold",
        "fontSize": 20,
        "fontStyle": "normal",
        "marginBottom": 10,
        "marginTop": 40,
        "textAlign": "'left'",
      }
    }
  >
    MOCKED_STRING:profile:favoriteGenres
  </Text>
  <View
    style={{}}
  >
    <View
      style={
        {
          "alignItems": "center",
          "flexDirection": "row",
          "flexWrap": "wrap",
          "justifyContent": "space-between",
          "width": "100%",
        }
      }
    >
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            {
              "alignItems": "center",
              "backgroundColor": "#28292E",
              "borderBottomLeftRadius": 8,
              "borderBottomRightRadius": 8,
              "borderTopLeftRadius": 8,
              "borderTopRightRadius": 8,
              "height": 215.625,
              "justifyContent": "center",
              "marginBottom": 15,
              "width": 345,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "backgroundColor": "#28292e",
                "borderBottomLeftRadius": 8,
                "borderBottomRightRadius": 8,
                "borderColor": "#313236",
                "borderStyle": "solid",
                "borderTopLeftRadius": 8,
                "borderTopRightRadius": 8,
                "borderWidth": 1,
                "height": "100%",
                "justifyContent": "center",
                "width": "100%",
              }
            }
          >
            <Image
              itemHeight={35}
              itemWidth={30}
              source={1}
              style={
                {
                  "height": 35,
                  "marginBottom": 10,
                  "paddingBottom": 15,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 15,
                  "width": 30,
                }
              }
            />
            <Text
              style={
                {
                  "color": "#cfcfd0",
                  "fontFamily": "Poppins-Regular",
                  "fontSize": 14,
                  "fontWeight": "400",
                }
              }
            >
              Action
            </Text>
          </View>
        </View>
      </View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            {
              "alignItems": "center",
              "backgroundColor": "#28292E",
              "borderBottomLeftRadius": 8,
              "borderBottomRightRadius": 8,
              "borderTopLeftRadius": 8,
              "borderTopRightRadius": 8,
              "height": 215.625,
              "justifyContent": "center",
              "marginBottom": 15,
              "width": 345,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "backgroundColor": "#28292e",
                "borderBottomLeftRadius": 8,
                "borderBottomRightRadius": 8,
                "borderColor": "#313236",
                "borderStyle": "solid",
                "borderTopLeftRadius": 8,
                "borderTopRightRadius": 8,
                "borderWidth": 1,
                "height": "100%",
                "justifyContent": "center",
                "width": "100%",
              }
            }
          >
            <Image
              itemHeight={30}
              itemWidth={32}
              source={1}
              style={
                {
                  "height": 30,
                  "marginBottom": 10,
                  "paddingBottom": 15,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 15,
                  "width": 32,
                }
              }
            />
            <Text
              style={
                {
                  "color": "#cfcfd0",
                  "fontFamily": "Poppins-Regular",
                  "fontSize": 14,
                  "fontWeight": "400",
                }
              }
            >
              Adventure
            </Text>
          </View>
        </View>
      </View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            {
              "alignItems": "center",
              "backgroundColor": "#28292E",
              "borderBottomLeftRadius": 8,
              "borderBottomRightRadius": 8,
              "borderTopLeftRadius": 8,
              "borderTopRightRadius": 8,
              "height": 215.625,
              "justifyContent": "center",
              "marginBottom": 15,
              "width": 345,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "backgroundColor": "#28292e",
                "borderBottomLeftRadius": 8,
                "borderBottomRightRadius": 8,
                "borderColor": "#313236",
                "borderStyle": "solid",
                "borderTopLeftRadius": 8,
                "borderTopRightRadius": 8,
                "borderWidth": 1,
                "height": "100%",
                "justifyContent": "center",
                "width": "100%",
              }
            }
          >
            <Image
              itemHeight={30}
              itemWidth={29}
              source={1}
              style={
                {
                  "height": 30,
                  "marginBottom": 10,
                  "paddingBottom": 15,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 15,
                  "width": 29,
                }
              }
            />
            <Text
              style={
                {
                  "color": "#cfcfd0",
                  "fontFamily": "Poppins-Regular",
                  "fontSize": 14,
                  "fontWeight": "400",
                }
              }
            >
              Casual
            </Text>
          </View>
        </View>
      </View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            {
              "alignItems": "center",
              "backgroundColor": "#28292E",
              "borderBottomLeftRadius": 8,
              "borderBottomRightRadius": 8,
              "borderTopLeftRadius": 8,
              "borderTopRightRadius": 8,
              "height": 215.625,
              "justifyContent": "center",
              "marginBottom": 15,
              "width": 345,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "backgroundColor": "#28292e",
                "borderBottomLeftRadius": 8,
                "borderBottomRightRadius": 8,
                "borderColor": "#313236",
                "borderStyle": "solid",
                "borderTopLeftRadius": 8,
                "borderTopRightRadius": 8,
                "borderWidth": 1,
                "height": "100%",
                "justifyContent": "center",
                "width": "100%",
              }
            }
          >
            <Image
              itemHeight={31}
              itemWidth={10}
              source={1}
              style={
                {
                  "height": 31,
                  "marginBottom": 10,
                  "paddingBottom": 15,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 15,
                  "width": 10,
                }
              }
            />
            <Text
              style={
                {
                  "color": "#cfcfd0",
                  "fontFamily": "Poppins-Regular",
                  "fontSize": 14,
                  "fontWeight": "400",
                }
              }
            >
              Fighting
            </Text>
          </View>
        </View>
      </View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            {
              "alignItems": "center",
              "backgroundColor": "#28292E",
              "borderBottomLeftRadius": 8,
              "borderBottomRightRadius": 8,
              "borderTopLeftRadius": 8,
              "borderTopRightRadius": 8,
              "height": 215.625,
              "justifyContent": "center",
              "marginBottom": 15,
              "width": 345,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "backgroundColor": "#28292e",
                "borderBottomLeftRadius": 8,
                "borderBottomRightRadius": 8,
                "borderColor": "#313236",
                "borderStyle": "solid",
                "borderTopLeftRadius": 8,
                "borderTopRightRadius": 8,
                "borderWidth": 1,
                "height": "100%",
                "justifyContent": "center",
                "width": "100%",
              }
            }
          >
            <Image
              itemHeight={42}
              itemWidth={22}
              source={1}
              style={
                {
                  "height": 42,
                  "marginBottom": 10,
                  "paddingBottom": 15,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 15,
                  "width": 22,
                }
              }
            />
            <Text
              style={
                {
                  "color": "#cfcfd0",
                  "fontFamily": "Poppins-Regular",
                  "fontSize": 14,
                  "fontWeight": "400",
                }
              }
            >
              Music & Party
            </Text>
          </View>
        </View>
      </View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            {
              "alignItems": "center",
              "backgroundColor": "#28292E",
              "borderBottomLeftRadius": 8,
              "borderBottomRightRadius": 8,
              "borderTopLeftRadius": 8,
              "borderTopRightRadius": 8,
              "height": 215.625,
              "justifyContent": "center",
              "marginBottom": 15,
              "width": 345,
            }
          }
        >
          <ViewManagerAdapter_ExpoLinearGradient
            colors={
              [
                "#873FBB",
                "#C47AF5",
              ]
            }
            end={
              {
                "x": 1,
                "y": 0,
              }
            }
            start={
              {
                "x": 0,
                "y": 1,
              }
            }
            style={
              {
                "alignItems": "center",
                "borderBottomLeftRadius": 8,
                "borderBottomRightRadius": 8,
                "borderTopLeftRadius": 8,
                "borderTopRightRadius": 8,
                "height": "100%",
                "justifyContent": "center",
                "width": "100%",
              }
            }
          >
            <Image
              itemHeight={41}
              itemWidth={40}
              source={1}
              style={
                {
                  "height": 41,
                  "marginBottom": 10,
                  "paddingBottom": 15,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 15,
                  "width": 40,
                }
              }
            />
            <Text
              isSelected={true}
              style={
                {
                  "color": "#FFFFFF",
                  "fontFamily": "Poppins-Regular",
                  "fontSize": 14,
                  "fontWeight": "400",
                }
              }
            >
              RPG
            </Text>
          </ViewManagerAdapter_ExpoLinearGradient>
        </View>
      </View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            {
              "alignItems": "center",
              "backgroundColor": "#28292E",
              "borderBottomLeftRadius": 8,
              "borderBottomRightRadius": 8,
              "borderTopLeftRadius": 8,
              "borderTopRightRadius": 8,
              "height": 215.625,
              "justifyContent": "center",
              "marginBottom": 15,
              "width": 345,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "backgroundColor": "#28292e",
                "borderBottomLeftRadius": 8,
                "borderBottomRightRadius": 8,
                "borderColor": "#313236",
                "borderStyle": "solid",
                "borderTopLeftRadius": 8,
                "borderTopRightRadius": 8,
                "borderWidth": 1,
                "height": "100%",
                "justifyContent": "center",
                "width": "100%",
              }
            }
          >
            <Image
              itemHeight={30}
              itemWidth={30}
              source={1}
              style={
                {
                  "height": 30,
                  "marginBottom": 10,
                  "paddingBottom": 15,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 15,
                  "width": 30,
                }
              }
            />
            <Text
              style={
                {
                  "color": "#cfcfd0",
                  "fontFamily": "Poppins-Regular",
                  "fontSize": 14,
                  "fontWeight": "400",
                }
              }
            >
              Sports
            </Text>
          </View>
        </View>
      </View>
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            {
              "alignItems": "center",
              "backgroundColor": "#28292E",
              "borderBottomLeftRadius": 8,
              "borderBottomRightRadius": 8,
              "borderTopLeftRadius": 8,
              "borderTopRightRadius": 8,
              "height": 215.625,
              "justifyContent": "center",
              "marginBottom": 15,
              "width": 345,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "backgroundColor": "#28292e",
                "borderBottomLeftRadius": 8,
                "borderBottomRightRadius": 8,
                "borderColor": "#313236",
                "borderStyle": "solid",
                "borderTopLeftRadius": 8,
                "borderTopRightRadius": 8,
                "borderWidth": 1,
                "height": "100%",
                "justifyContent": "center",
                "width": "100%",
              }
            }
          >
            <Image
              itemHeight={38}
              itemWidth={40}
              source={1}
              style={
                {
                  "height": 38,
                  "marginBottom": 10,
                  "paddingBottom": 15,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 15,
                  "width": 40,
                }
              }
            />
            <Text
              style={
                {
                  "color": "#cfcfd0",
                  "fontFamily": "Poppins-Regular",
                  "fontSize": 14,
                  "fontWeight": "400",
                }
              }
            >
              Strategy
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
